x/payments/spec/03_messages.md

```diff
@@ -0,0 +1,145 @@
+# Messages
+
+In this section we describe the processing of the payment messages and the corresponding updates to the state. All created/modified state objects specified by each message are defined within the [state](./02_state.md) section.
+
+## MsgCreatePaymentTemplate
+
+PaymentTemplate can be created by any address using `MsgCreatePaymentTemplate`.
+
+| **Field**              | **Type**           | **Description**                                                                                               |
```

Comment:
Types should be actual types, e.g. string, bool, etc.

Reply:
modified